src/sidebar/components/menu.js

```diff
@@ -73,13 +75,17 @@ export default function Menu({
     if (event.type === 'mousedown') {
       ignoreNextClick = true;
     } else if (event.type === 'click' && ignoreNextClick) {
+      // ignore mouseup event
```

Comment:
I presume this comment means "ignore `click` event _caused_ by a mouse release", rather than an actual `mouseup` event which is a distinct DOM event?

Reply:
yep, I'll make that more clear